python/ray/train/examples/jax_mnist_example.py

```diff
@@ -0,0 +1,222 @@
+import argparse
+import functools
+import time
+from typing import Dict
+
+import einops
+import jax
+import jax.numpy as jnp
```

Comment:
should jax imports be removed from here?

Reply:
no problem